src/RestApi/StoreApi/Schemas/AbstractSchema.php

```diff
@@ -63,4 +63,23 @@ function( $property ) {
 			$properties
 		);
 	}
+
+	/**
+	 * Convert monetary values from WooCommerce to integers, using the smallest
+	 * unit of a currency.
```

Comment:
Slight change needed here since the function is returning strings right? So... "Convert monetary values from WooCommerce to strings..."?